lucene/core/src/test/org/apache/lucene/codecs/lucene99/TestGroupVInt.java

```diff
@@ -31,9 +34,7 @@ public void testEncodeDecode() throws IOException {
     long[] values = new long[ForUtil.BLOCK_SIZE];
     long[] restored = new long[ForUtil.BLOCK_SIZE];
     final int iterations = atLeast(100);
-
-    final GroupVIntWriter w = new GroupVIntWriter();
-    byte[] encoded = new byte[(int) (Integer.BYTES * ForUtil.BLOCK_SIZE * 1.25)];
+    Directory dir = FSDirectory.open(createTempDir());
```

Comment:
Let's use `newFSDirectory` to add coverage for all Directory implementations?

```suggestion
    Directory dir = newFSDirectory(createTempDir());
```

Reply:
+1